### PR TITLE
Removed variant IDs

### PR DESCRIPTION
**Pull request by seijisakurai**

### CODE REVIEW

File: ttps/credential-access/6044fb78-5d7b-43cc-9592-dcba3c4537f1.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ technique:
 platforms:
   darwin:
     sh:
-      command: dscl . read #{discovery.local.existing.user} ShadowHashData
+      command: dscl . read
```
Comment (privateducky) [inline ttps/credential-access/6044fb78-5d7b-43cc-9592-dcba3c4537f1.yml:12]: why is this ttp different?

File: ttps/discovery/27b922c1-76ca-41c7-926f-1c36cabec122.yml [hunk 2]
```diff
@@ -16,8 +16,7 @@ platforms:
     cmd:
       command: setspn.exe -Q */*
       variants:
-      - command: setspn.exe -T #{domain.T1018} -Q */*
```
Comment (privateducky) [inline ttps/discovery/27b922c1-76ca-41c7-926f-1c36cabec122.yml:19]: another different TTP

**Review by privateducky — CHANGES_REQUESTED** [review]
stopping my review at 2 TTPs: there's more here than just ID removals, so do a pass through and see if you can catch any non ID changes

**Comment by seijisakurai** [comment]
It seems like ruamel doesn't like it when you introduce hashtags that aren't double quoted. This is evident from the fact that double quoted facts don't break the unmarshalling. I will manually fix these.

**Review by privateducky — APPROVED** [review]

**Pull request merged**